packages/jest-message-util/src/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const {
+  formatExecError,
+  formatResultsErrors,
+  formatStackTrace,
+} = require('./messages');
```

Comment:
in a follow-up, can you simply move the contents of the messages file into this file? I don't think there is any point now in keeping them separate.